priv/repo/migrations/20210120134552_add_tokens_badge.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Safira.Repo.Migrations.AddTokensBadge do
+  use Ecto.Migration
+
+  def change do
+
+    alter table(:badges) do
+      add :tokens, :integer
+  end
+end
+end
```

Comment:
You should run `mix format priv/repo/migrations/20210120134552_add_tokens_badge.exs`